internal/api/models/converter.go

```diff
@@ -9,14 +9,15 @@ import (
 // This excludes sensitive fields like RequestedBy.
 func ToUserMediaItem(m database.Media, cfg *config.Config) UserMediaItem {
 	item := UserMediaItem{
-		ID:              m.ID,
-		Title:           m.Title,
-		Year:            m.Year,
-		MediaType:       MediaType(m.MediaType),
-		LibraryName:     m.LibraryName,
-		FileSize:        m.FileSize,
-		DefaultDeleteAt: m.DefaultDeleteAt,
-		Unkeepable:      m.Unkeepable,
+		ID:                m.ID,
+		Title:             m.Title,
+		Year:              m.Year,
+		MediaType:         MediaType(m.MediaType),
+		LibraryName:       m.LibraryName,
+		FileSize:          m.FileSize,
+		DefaultDeleteAt:   m.DefaultDeleteAt,
```

Comment:
It would be nice if `EstimatedDeleteAt` defaults to ` DefaultDeleteAt` in case the estimated delete time is zero. Otherwise the UI will show something like "Deletes 739603 days ago " until the scheduled job runs and calculates the estimated delete time.